addons/project/views/project_views.xml

```diff
@@ -19,11 +19,11 @@
                     <field name="name" string="Task" filter_domain="['|', ('name', 'ilike', self), ('id', 'ilike', self)]"/>
                     <field name="tag_ids"/>
                     <field name="user_ids" filter_domain="[('user_ids', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
-                    <field string="Project" name="display_project_id"/>
+                    <field string="Project" name="display_project_id" invisible="context.get('default_project_id', False)"/>
```

Comment:
I would rather like to create a primary inherited search view that would be used in "My Tasks" where the display_project_id field would be added.
This is a bit too hackish for me.

After having read the diff a bit more, shouldn't we just move it into `view_task_search_form_extended`? Isn't it the right one? I haven't tested though, could you check?